### PR TITLE
#333 App dependencies update

### PR DESCRIPTION
**Pull request by adamdavies1**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Please also include any acceptance criteria if you have any.

- Upgrade designer dependencies 
- Upgraded model dependencies
- Removed some dependencies in the root package.json
[Model failed packages.docx](https://github.com/XGovFormBuilder/digital-form-builder/files/8216276/Model.failed.packages.docx)
[Designer Failed Packages update List.docx](https://github.com/XGovFormBuilder/digital-form-builder/files/8216279/Designer.Failed.Packages.update.List.docx)
[Smoke test update failures.docx](https://github.com/XGovFormBuilder/digital-form-builder/files/8325840/Smoke.test.update.failures.docx)
[Runner__Failed_Packages.docx](https://github.com/XGovFormBuilder/digital-form-builder/files/8340974/Runner__Failed_Packages.docx)

Some of these packages when upgrading caused huge build issues.  We have create a list for each envrioment so these can be solved at a later date.

Also it seems that some root packages that are duplicated in to other environments cant be removed without cause wither build or pipeline failures (lint for example).

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce
the testing if necessary.

Before PR's can be merged they will need to be tested by QA and approved where
applicable. To flag the change to QA assign **@XGovFormBuilder/qa** as one of the reviewers.

- [ ] Run, build, test and run smoke tests

# Checklist:

- [x] My changes do not introduce any new linting errors
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I have rebased onto main and there are no code conflicts
- [x] I have checked deployments are working in all environments

### CODE REVIEW

**Comment by adamdavies1** [comment]
[Designer Failed Packages update List.docx](https://github.com/XGovFormBuilder/digital-form-builder/files/8216270/Designer.Failed.Packages.update.List.docx)
[Model failed packages.docx](https://github.com/XGovFormBuilder/digital-form-builder/files/8216272/Model.failed.packages.docx)

**Review by jenbutongit — CHANGES_REQUESTED** [review]
just checking to see if this

File: .github/workflows/branch--lint-unit-and-smoke-test.yml [hunk 1]
```diff
@@ -5,8 +5,8 @@ on:
       - master
       - main
     paths-ignore:
-      - 'docs/**'
-      - '**/README.md'
+      - "docs/**"
```
[the remark this answers is not in the document]
Comment (adamdavies1) [reply .github/workflows/branch--lint-unit-and-smoke-test.yml]: Just changed this back, it hasnt triggered the build

File: .github/workflows/main--lint-unit-build-and-publish-images.yml [hunk 2]
```diff
@@ -121,13 +121,6 @@ jobs:
         # if you change this, please remember to apply the same changes to branch--lint-unit-and-smoke-test.yml
         # so main branch cache can be shared with branches
         run: echo "DAYOFYEAR=$(date +%j)" >> $GITHUB_ENV
-      - name: Docker layer caching
```
Comment (adamdavies1) [inline .github/workflows/main--lint-unit-build-and-publish-images.yml:124]: This is in place to fix the pipeline cache when it fills up and causes a out of memory failure

File: docker-compose.yml [hunk 3]
```diff
@@ -8,6 +8,8 @@ services:
     build:
       context: .
       dockerfile: ./designer/Dockerfile
+      cache_from:
```
Comment (adamdavies1) [inline docker-compose.yml:11]: This is in place to fix the pipeline cache when it fills up and causes a out of memory failure

File: smoke-tests/designer/wdio.conf.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 const { hooks } = require("./support/hooks");
 const drivers = {
-  chrome: { version: "97.0.4692.71" }, // https://chromedriver.chromium.org/
+  chrome: { version: "98.0.4758.102" }, // https://chromedriver.chromium.org/
```
Comment (adamdavies1) [inline smoke-tests/designer/wdio.conf.js:3]: fixes chrome driver out of date error

**Review by jenbutongit — APPROVED** [review]

**Comment by jenbutongit** [comment]
@adamdavies1 - on smoke tests.. don't worry. I'm moving those over to cypress anyway (see #703)

**Review by jenbutongit — CHANGES_REQUESTED** [review]
LGTM - I think it needs a conflict resolve though

**Review by jenbutongit — APPROVED** [review]

**Pull request merged**